src/Raven.Studio/wwwroot/App/views/shell/about.html

```diff
@@ -139,22 +139,33 @@ <h4>Security</h4>
             <div class="padding">
                 <div class="license-actions flex-horizontal">
                     <!-- ko if: !registered() -->
-                    <button class="btn btn-primary flex-grow" data-bind="click: register" title="Register a new license">REGISTER LICENSE</button>
+                    <button class="btn btn-primary flex-grow" 
+                            data-bind="click: register, 
+                                       enable: activateLicenseConfigurationSettings,
+                                       attr: { title: activateLicenseConfigurationSettings() ? 'Register a new license' : 'Registering new license is disabled in the server configuration' }"> 
```

Comment:
please make sure title works when button is disabled

Reply:
done